sway/config.c

```diff
@@ -404,7 +404,7 @@ static bool load_config(const char *path, struct sway_config *config,
 		sway_log(SWAY_ERROR, "Error(s) loading config!");
 	}
 
-	return true;
+	return config->active || !config->validating || config_load_success;
```

Comment:
What's the reason why we ignore errors when not validating or when the config is already active?

Do we want not to make the `reoad` command fail on error?

Reply:
I just left it at the current behavior for every case other than validating only. The current config options don't really need to be fatal. The errors just show in swaynag. The fatal ones would be the currently not implemented security ones.

Reply:
The return value seems to just passed through down to the `reload` command handler. I don't know whether i3 makes the `reload` command fail on config error.

In any case, I agree the current patch is enough to fix the original issue.